spec-main/api-tray-spec.ts

```diff
@@ -1,42 +1,41 @@
-const { remote } = require('electron')
-const { expect } = require('chai')
-const { Menu, Tray, nativeImage } = remote
+import { expect } from 'chai'
+import { Menu, Tray, nativeImage } from 'electron'
 
 describe('tray module', () => {
-  let tray
+  let tray: Tray | null = null
```

Comment:
I'd prefer to do
```suggestion
  let tray: Tray;
```

Then do a workaround in `afterEach` to set it to null rather than have to use `!` everywhere 🤔

Reply:
oki done sth - imo a bit hacky, but works 👀